package.json

```
@@ -83,10 +83,10 @@
     "format": "eslint --ignore-path .gitignore --ext .js,.jsx --fix .",
     "clean": "if [ -d lib ]; then rm -r lib; fi",
     "prebuild": "npm run clean",
-    "build:js": "babel src --out-dir lib --verbose",
+    "build:js": "babel src --out-dir \"${BUILD_PATH}lib\" --verbose",
     "build:css": "run-p 'build:css:* -- {@}' --",
-    "build:css:local": "postcss 'src/**/*.css' --base src --dir lib --verbose",
-    "build:css:global": "postcss 'src/**/*.css' --base src --env=global --dir lib/global --verbose",
+    "build:css:local": "postcss 'src/**/*.css' --base src --dir \"${BUILD_PATH}lib\" --verbose",
```

@Joozty I can make this a bit safer disabling it in production but it looks ugly and too hard to understand, in my opinion.

```suggestion
    "build:css:local": "postcss 'src/**/*.css' --base src --dir \"$( [ \\\"$NODE_ENV\\\" = 'production' ] && echo '' || echo $BUILD_PATH )lib\" --verbose",
```

---

Rather not, it's really ugly 😄